src/compiler/checker.ts

```diff
@@ -14996,7 +14996,7 @@ namespace ts {
                         links.resolvedType = getTypeFromTypeNode(node.type);
                         break;
                     default:
-                        throw Debug.assertNever(node.operator);
+                        throw new Error("never");

```

Comment:
You shouldn't need to make this change. `assertNever` is here to ensure all possible cases for `node.operator` are handled.